package.json

```diff
@@ -15,12 +15,15 @@
     "eslint-plugin-jest": "^23.6.0",
     "eslint-plugin-simple-import-sort": "^5.0.1",
     "eslint-plugin-unicorn": "^16.0.0",
+    "jest": "^25.1.0",
+    "ts-jest": "^25.0.0",
```

Comment:
Those are definitely not `dependencies`.

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/267a96360f447175bf63c69e8f81d71b5db5c652